app/src/main/java/org/kiwix/kiwixmobile/webserver/WebServerHelper.java

```diff
@@ -0,0 +1,183 @@
+package org.kiwix.kiwixmobile.webserver;
```

Comment:
We don't need this file as it will be handled by kiwix-lib. You can move this to some other branch if you need it for later use.

Reply:
Fixed 062dd6211419186e7e104f3282fe7982a086aed4